README.md

```diff
@@ -9,7 +9,7 @@ creating languages and generating efficient compilers.
 Before you test the Miking system, you need to install
```

Comment:
I have rewritten this part in PR #107. I would prefer the other changes.

Reply:
Hence, it would be good if you reverted this sentence, so that we can rebase without conflicts.

Reply:
Done